dev/com.ibm.ws.webcontainer.servlet.3.1/src/com/ibm/ws/webcontainer31/osgi/srt/SRTConnectionContext31.java

```diff
@@ -135,6 +137,7 @@ public void finishConnection()
                             // remove the TransportConstants which if added for Upgrade previously
                             vc.getStateMap().put(TransportConstants.CLOSE_UPGRADED_WEBCONNECTION, null);
                             vc.getStateMap().put(TransportConstants.UPGRADED_LISTENER, null);
+                            vc.getStateMap().put(TransportConstants.ON_CLOSE_COUNTDOWN_LATCH, null);
```

Comment:
Not sure if I should this, but it seemed need to keep it consistent with the puts on lines 110-111.